#!/usr/bin/env python
PACKAGE = "map_transform"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("obs_index", int_t, 0, "Obstacle Index to debug Importance", 0, 0,100)

exit(gen.generate(PACKAGE, "map-planner", "ImportanceMapPlanner"))
